libraries/botbuilder-core/src/testAdapter.ts

```diff
@@ -482,13 +482,17 @@ export class TestFlow {
      * @param description (Optional) Description of the test case. If not provided one will be generated.
      * @param timeout (Optional) number of milliseconds to wait for a response from bot. Defaults to a value of `3000`.
      */
-    public assertReply(expected: string | Partial<Activity> | TestActivityInspector, description?: string, timeout?: number): TestFlow {
+    public assertReply(expected: string | Partial<Activity> | TestActivityInspector, description?: string, timeout?: number, isSubstring: boolean = false): TestFlow {
```

Comment:
I'm probably missing the obvious. Can you help me understand? 

The C# has the following signature:
```cs
public TestFlow AssertReplyContains(string expected, string description = null, uint timeout = 3000)
```

Why would the JS have a different signature?